www/pages/my-buses/my-buses-controller.js

```diff
@@ -58,18 +58,9 @@ angular.module('pvta.controllers').controller('MyBusesController', function ($sc
   }
 
   var reload = function () {
-    localforage.getItem('updatedRoutes', function (err, updated) {
```

Comment:
Since this PR adds a new migration that just tosses everything, I figured there's no point in leaving this one here.